src/seedu/addressbook/commands/fees/AddFeesCommand.java

```diff
@@ -47,6 +51,16 @@ public CommandResult execute() {
             final ReadOnlyPerson target = getTargetReadOnlyPerson();
```

Comment:
One of the changes that has been implemented in a previous patch is that getTargetPerson returns an editable Person, while getTargetReadOnlyPerson returns uneditable one now. Since the commands are already distinguished, we need not use both functions anymore. Can directly do Person person = getTargetPerson(); if you want the editable version.

Reply:
AWESOME. Thanks I missed that